src/gologin-api.js

```diff
@@ -0,0 +1,82 @@
+import puppeteer from 'puppeteer-core';
+import GoLogin from './gologin.js';
+
+export function getDefaultParams() {
+  return {
+    token: process.env.GL_API_TOKEN,
+    profile_id: process.env.GL_PROFILE_ID,
+    executablePath: process.env.GL_EXECUTABLE_PATH,
+  };
+}
+
+const createLegacyGologin = (params) => {
+  const defaults = getDefaultParams();
+  return new GoLogin({
+    ...defaults,
+    ...params,
+  });
+};
+
+export const delay = (ms = 250) => new Promise((res) => setTimeout(res, ms));
+
+export function GologinApi({ token }) {
+  if (!token) {
+    throw new Error('GoLogin API token is missing');
+  }
+  const browsers = [];
+  const legacyGls = [];
```

Comment:
using `browsers` and `legacyGIs` this way we introduce issues with browsers close
`process.exit()` will instantly kill main process and clean up references to open browsers making unclosed processes leaving forever
this way, if user has 2 or more `GologinApi` instances, exit command will properly close only 1 browser, but others will stay open

please, move `browsers` and `legacyGIs` out of `GologinApi` class to avoid this problem